src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { ErrorMessages } from '../../types/ErrorMessages';
+
+type Props = {
+  typeOfError: ErrorMessages,
+  setMessage: () => void,
+};
+
+export const ErrorMessage: React.FC<Props> = ({ typeOfError, setMessage }) => {
+  return (
+    <div className="notification is-danger is-light has-text-weight-normal">
+      <button
+        type="button"
+        className="delete"
+        onClick={setMessage}
+        aria-label="Close error"
+      />
+
+      {typeOfError === ErrorMessages.LOAD && (
+        'Unable to load todos'
+      )}
+
+      {typeOfError === ErrorMessages.TITLE && (
+        'Title can\'t be empty'
+      )}
+
+      {typeOfError === ErrorMessages.ADD && (
+        'Unable to add the todo'
+      )}
```

Comment:
such  code definitely cannot get into the master.
 Can make a function with a switch case that will determine exactly which error needs to be shown and transfer a ready-made message to this component